src/components/Loading/Loading.scss

```diff
@@ -11,14 +17,13 @@
 
   // gray-dark #43484e in hex.
   --faded-gray-dark: 67, 72, 78;
-  background-color: rgba(var(--faded-gray-dark), 0.75);
-  // TODO: The above can be replaced with the below when Firefox supports "Relative RGB colors".
-  //background-color: rgb(from var(--gray-dark) / 0.75);
+
+  background-color: rgb(from var(--gray-dark) / 75%);
```

Comment:
The loading overlay is missing with this change. I think it's supposed to be
`
background-color: rgb(var(--faded-gray-dark), 0.75);
`

Reply:
These were updated automatically, but  I changed them based on your comments